contracts/tokenomics/generator/src/contract.rs

```diff
@@ -2521,16 +2657,66 @@ pub fn migrate(mut deps: DepsMut, env: Env, msg: MigrateMsg) -> Result<Response,
                         has_asset_rewards: pool_info_v120.has_asset_rewards,
                         accumulated_proxy_rewards_per_share,
                         orphan_proxy_rewards,
-                        accumulated_rewards_per_share: pool_info_v120.accumulated_rewards_per_share,
                         last_reward_block: pool_info_v120.last_reward_block,
                         proxy_reward_balance_before_update: pool_info_v120
                             .proxy_reward_balance_before_update,
                         reward_proxy: pool_info_v120.reward_proxy,
+                        reward_global_index: pool_info_v120.reward_global_index,
+                        total_virtual_supply: pool_info_v120.total_virtual_supply,
                     };
                     POOL_INFO.save(deps.storage, &Addr::unchecked(key), &pool_info)?;
                 }
                 migration::migrate_configs_to_v130(deps.storage)?;
             }
+            "1.3.0" => {
+                let msg: migration::MigrationMsgV140 = from_binary(&msg.params)?;
+
+                let keys = POOL_INFO
+                    .keys(deps.storage, None, None, cosmwasm_std::Order::Ascending {})
+                    .map(|v| String::from_utf8(v).map_err(StdError::from))
+                    .collect::<Result<Vec<String>, StdError>>()?;
+
+                for key in keys {
+                    let pool_info_v130 = migration::POOL_INFOV130
+                        .load(deps.storage, &Addr::unchecked(key.clone()))?;
+
+                    let mut accumulated_proxy_rewards_per_share = Default::default();
```

Comment:
We must not migrate all this proxy related stuff since it should be already set in v.1.3.0.